modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/ToolbarDrawerToggleTest.ts

```diff
@@ -68,4 +69,43 @@ describe('browser.tinymce.themes.silver.editor.toolbar.ToolbarDrawerToggleTest',
       pTestToggle({ toolbar: [ 'undo redo', 'bold italic' ], inline: true }, false)
     );
   });
+
+  const pTestEvent = async (toolbarMode: ToolbarMode, command: (editor: Editor) => void) => {
+    const editor = await McEditor.pFromSettings<Editor>({
+      menubar: false,
+      statusbar: false,
+      width: 200,
+      toolbar_mode: toolbarMode,
+      base_url: '/project/tinymce/js/tinymce'
+    });
+    editor.focus();
+
+    const lastState = Singleton.value<boolean>();
+    lastState.set(false);
```

Comment:
By using a single value here, we aren't checking how many times the event is firing ... just what the last value was. Is that a problem? Should we consider using something closer to an array like TestStore?

Reply:
Yeah, probably counting the amount of triggered events is a good idea. Updated with `TestStore` [here](https://github.com/tinymce/tinymce/pull/8238/commits/a900ff7e91cd86ca15912a4a076d09a1d12af975).